ui/raidboss/data/03-hw/raid/a12n.js

```diff
@@ -0,0 +1,371 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.AlexanderTheSoulOfTheCreator,
+  timelineFile: 'a12n.txt',
+  timelineTriggers: [
+    {
+      id: 'A12N Tank Limit Break',
+      regex: /Divine Judgment/,
+      beforeSeconds: 5,
+      condition: function(data) {
+        // Let's not spam tanks if they are unlucky enough to see enrage.
+        return !data.judgment && data.role == 'tank';
+      },
+      alarmText: {
+        en: 'Limit break now!',
+      },
+      run: function(data) {
+        data.judgment = true;
```

Comment:
Data is cleared between pulls, so this works.

Two other options you could choose that would eliminate the need for a variable:
1) `suppressSeconds: 9999`.
2) `regex: /^Divine Judgment$/,`

Reply:
Ah, option 1 is definitely just what we want here.

Reply:
Could you also delete the `run` section too?